packages/run-protocol/CHANGELOG.md

```diff
@@ -3,6 +3,79 @@
+* **run-protocol:** more support for governance boot ([711d80d](https://github.com/Agoric/agoric-sdk/commit/711d80d6f4b854ca9dadb0bae764a9a0cc65fa59))
+* **run-protocol:** re-structure vault notifiers to work with wallet ([9ac3f00](https://github.com/Agoric/agoric-sdk/commit/9ac3f00462cff6cfc20ab3325dad6798f3a8560f))
+* **run-protocol:** shuffle around to fix types ([1c06bbd](https://github.com/Agoric/agoric-sdk/commit/1c06bbd71c39b09bb0e8007b0a96febf3bfbd771))
+* **run-protocol:** store Presences, not Promises, in VaultManager ([5aee8af](https://github.com/Agoric/agoric-sdk/commit/5aee8af34fb1fab54633fc9a1acbf2818414de9a)), closes [#5106](https://github.com/Agoric/agoric-sdk/issues/5106) [#5121](https://github.com/Agoric/agoric-sdk/issues/5121) [#5106](https://github.com/Agoric/agoric-sdk/issues/5106)
+* **run-protocol:** use wallet-friendly offer result notifiers ([b08330b](https://github.com/Agoric/agoric-sdk/commit/b08330b5bbb040979a68c19c8609e715e468b905))
+* **vaults:** check args before acting in addVaultType ([12d5cfb](https://github.com/Agoric/agoric-sdk/commit/12d5cfbc9abfa553e40b7b458ce99420c7c54a85))
+* Encode Passables, not just keys ([#4470](https://github.com/Agoric/agoric-sdk/issues/4470)) ([715950d](https://github.com/Agoric/agoric-sdk/commit/715950d6bfcbe6bc778b65a256dc5d26299172db))
+* fix a bug in addLiquidity ([#5091](https://github.com/Agoric/agoric-sdk/issues/5091)) ([512db54](https://github.com/Agoric/agoric-sdk/commit/512db545c4e88fa4126c44a29f3a8775c330264f))
+* renamings [#4470](https://github.com/Agoric/agoric-sdk/issues/4470) missed ([#4896](https://github.com/Agoric/agoric-sdk/issues/4896)) ([98c9f0e](https://github.com/Agoric/agoric-sdk/commit/98c9f0eabf6f0a85581e34ca0adf24f9805d1f0c))
+* two isolated cases where a missing argument did not default ([531d367](https://github.com/Agoric/agoric-sdk/commit/531d367600e97652babff1ee8ffa4e4665f50baa))
+* update types to specify ERef<Issuer> on addPool() ([6c13d99](https://github.com/Agoric/agoric-sdk/commit/6c13d997f89d914516dd6d4821d95364bd715b39)), closes [#4810](https://github.com/Agoric/agoric-sdk/issues/4810)
+* **vats:** move `startPriceAuthority` earlier in the boot sequence ([bf93171](https://github.com/Agoric/agoric-sdk/commit/bf93171c69eb1a19b04c24c9283e0d433ca9d411))
+* **vault:** make vault transfer invitation legible ([#4844](https://github.com/Agoric/agoric-sdk/issues/4844)) ([325ef39](https://github.com/Agoric/agoric-sdk/commit/325ef399cc9b65eedca71c2d2d7c42a4c6ec5191))
+* **zoe:** pass brands (not issuers) to priceAggregator ([5800711](https://github.com/Agoric/agoric-sdk/commit/580071189bb60d83ceaa806bf85035173ae9563c))
+
+
+### Reverts
+
+* Revert "refactor(run-protocol): virtual kind for vault manager (#5052)" ([b08dc58](https://github.com/Agoric/agoric-sdk/commit/b08dc5836e8bea98de4316edc7ac5eef698c7072)), closes [#5052](https://github.com/Agoric/agoric-sdk/issues/5052)
+
+
```

Comment:
this was unreverted (only reverted to unblock CI). It's confusing to have in the changelog and I think we should strike it.